README.md

```diff
@@ -127,7 +122,7 @@ Since we are `link`ing the examples' dependency on `reason-urql` to the `src` di
 
 ## Getting Involved
 
-This project is currently under active development. Please help out by [opening an issue](https://github.com/FormidableLabs/reason-urql/issues) or [filing a PR](https://github.com/FormidableLabs/reason-urql/pulls).
+Please help out by [opening an issue](https://github.com/FormidableLabs/reason-urql/issues) or [filing a PR](https://github.com/FormidableLabs/reason-urql/pulls).
```

Comment:
This project still is under active development, but we at Formidable are sticking to using the Maintenance Status section as our source of truth for where maintenance is at and, since this is still experimental, I decided to remove this sentence here (but don't worry, I'm still making changes and continuing to bind new versions of `urql`).